### PR TITLE
Fix a few bugs in CIS for Ubuntu 22.04 

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- This PR fixes a few issues in CIS for Ubuntu 22.04, such as:
  - Improve Ubuntu OVAL regex for pam_faillock, this goes in hand to the previous commit 3cc26e85
  - Fix CIS reference for mount_option_home_nosuid
  - Add no_forward_files to profile, this was mistakenly never added before

#### Rationale:

- This is needed for CIS on Ubuntu 22.04

### CODE REVIEW

File: linux_os/guide/system/permissions/mounting/kernel_module_freevxfs_disabled/rule.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhcos4,rhel7,rhel8,rhel9,rhv4,sle15,ubuntu1804,ubuntu2004,ubuntu2204
+prodtype: fedora,ol7,ol8,rhcos4,rhel7,rhel8,rhel9,rhv4,sle15,ubuntu1804,ubuntu2004
```
Comment (marcusburghardt) [inline linux_os/guide/system/permissions/mounting/kernel_module_freevxfs_disabled/rule.yml:3]: Removing a product from `prodtype` might cause issues for users who used this rule in any tailored file. With this removal, the rule will no longer be present in the Datastream.
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#agreements

Considering this, is the removal indeed desired?

Comment (dodys) [reply linux_os/guide/system/permissions/mounting/kernel_module_freevxfs_disabled/rule.yml:3]: yes it is desired, that rule is not part of CIS for 22.04
unfortunately that was added as someone ported CIS for 20.04 to build on 22.04 before we had a proper CIS for 22.04. That won't happen in the future.

Comment (marcusburghardt) [reply linux_os/guide/system/permissions/mounting/kernel_module_freevxfs_disabled/rule.yml:3]: Ok. Thanks for confirming.

**Review by marcusburghardt — APPROVED** [review]

**Comment by marcusburghardt** [comment]
Overriding CODEOWNERS as @dodys can't approve his own PR.

**Pull request merged**